hoot-core-test/src/test/cpp/hoot/core/schema/ImplicitTagRawRulesDeriverTest.cpp

```diff
@@ -54,25 +54,9 @@ class ImplicitTagRawRulesDeriverTest : public CppUnit::TestFixture
   static QString inDir() { return "test-files/schema/ImplicitTagRawRulesDeriverTest"; }
   static QString outDir() { return "test-output/schema/ImplicitTagRawRulesDeriverTest"; }
 
-  // With parallel testing, these tests were all trying to create this same directory at once,
-  // failing occaisionally. This seems to help.
-  static void makeOutDir()
-  {
-    // Try twice, then die
-    if (!QDir().mkpath(outDir()))
-    {
-      sleep(100);
-      if (!QDir().mkpath(outDir()))
-      {
-        QString message = QString("Couldn't create output directory:") + outDir();
-        CPPUNIT_FAIL(message.toStdString());
-      }
-    }
-  }
-
```

Comment:
Thanks for the inspiration Micah but I'm deleting your function and moving it to `TestUtils`.

Reply:
Progress!!